app/index.css

```diff
@@ -1,10 +1,31 @@
+@font-face {
+  font-weight: 400;
+  font-family: "OpenSans";
+  src: url("./src/components/fonts/OpenSans-Medium.ttf") format("ttf");
+}
```

Comment:
probably dont needed here cause we loaded font from the internet in `index.html`
try to remove it